src/pe_source/data/pe_db/db_query_source.py

```diff
@@ -458,24 +462,63 @@ def insert_intelx_breaches(df):
     cursor.close()
 
 
-def get_intelx_breaches(source_uid):
-    """Get IntelX credential breaches."""
-    conn = connect()
+def get_intelx_breaches(source_uid, redo_interval=3):
+    """
+    Query API for all IntelX credential breaches.
+
+    Args:
+        source_uid: The data source uid to filter credential breaches by
```

Comment:
Can you add a line for the `redo_interval` argument?

Reply:
commit 8057968922b4f04a61c4e75e5f94c6abf26b65e1